frontend/TraderX/src/views/profile/index.vue

```diff
@@ -50,7 +50,7 @@
                 label="EditProfile"
                 name="editprofile"
               >
-                <editprofile />
+                <editprofile :user="this.user"/>
```

Comment:
To solve the problem related to following/follower counts in profile page, you may send request to server to update user in **created** function.

Reply:
Actually I found the problem but couldn't solve it. The problem is the following; we get the user info when we try to render the profile page. But before response comes from the backend, we also try to render the usercard component. Because of the latency, usercard component sees an undefined object and fails to render the true object. 

I think we can solve it together when we make our next meeting